app/views/admin/settings/_custom_css.haml

```diff
@@ -1,6 +1,6 @@
 - if admin_settings.using_custom_css?
-  = field_set_tag 'Custom CSS for OTML' do
+  = field_set_tag 'Custom CSS for Runnable (legacy)' do
     %code.tiny
       = admin_settings.custom_css
 - else
-  %b No Custom CSS for OTML
+  %b No Custom CSS for Runnable (legacy)
```

Comment:
Perhaps it is on your list to take care of later. But I think it would be good to remove this field from admin_settings completely.  It looks like the editing views of the field are removed as part of this PR.